libraries/liveIntentIdSystem/shared.js

```diff
@@ -0,0 +1,179 @@
+import {UID1_EIDS} from '../uid1Eids/uid1Eids.js';
```

Comment:
There is a notion of shared ID in Prebid. The name of this file might cause misunderstandings. I propose to rename it to `liveIntentIdSystemShared`.